docs/styles.scss

```diff
@@ -1,8 +1,13 @@
 $fa-font-path: '../node_modules/font-awesome/fonts';
-@import '../node_modules/font-awesome/scss/font-awesome';
-@import '../lib/src/colors';
-@import '../lib/src/tables.scss';
+@import 'font-awesome';
 
+$FontPathOpenSans: "../node_modules/npm-font-open-sans/fonts";
+@import 'open-sans.scss';
+
+@import './cashmere-import';
+// *, *::before, *::after {
```

Comment:
It might require some changes to the doc styles, but uncommenting the _reset could be a good way to ensure that the components appear the same way across all browsers.

Reply:
I'd spent too long on this already, but yes, if we want this reset in we'll have to go through and make changes to accommodate it. Same with the `box-sizing:border-box`.